docs/source/guides/common_instruments.rst

```diff
@@ -0,0 +1,397 @@
+.. _common_instruments:
+
+Common Instrument Patterns
+===========================
+
+This guide explains how to create "common" instrument packages for shared components across multiple beamline instruments, based on successful deployment patterns at APS.
+
+Quick Start: When to Use Common Packages
+-----------------------------------------
+
+**Use common packages when you have:**
+
+- Multiple endstations sharing hardware (slits, monochromators, detectors)
+- Standardized procedures across experimental techniques
+- Beamline-wide calibration and alignment protocols
+
+**Create a common package in 2 commands:**
+
+.. code-block:: bash
+
+    mkdir -p src/beamline_common/{devices,plans}
+    echo '"""Support for beamline instruments."""' > src/beamline_common/__init__.py
+
+**Use the common package:**
+
+.. code-block:: python
+
+    # In src/endstation_a/startup.py
+    from beamline_common.devices import SharedOptics
+    from beamline_common.plans import standard_alignment
+
+Complete Common Instrument Guide
+---------------------------------
+
+Understanding Common Package Architecture
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Common packages contain **reusable components** but are **not instruments themselves**:
+
+.. code-block:: text
+
+    multi-beamline-project/
+    ├── src/
+    │   ├── id12_common/              # Shared 12-ID components
+    │   │   ├── __init__.py           # "Support for 12-ID instruments"
+    │   │   ├── devices/              # Shared hardware definitions
+    │   │   │   ├── area_detector.py  # Common detector configurations
```

Comment:
Needs an `__init__.py` file at this level so these modules can be used in YML files (which relies on `importlib`).